docs/theme/mlflow/static/css/simple-cards.css

```diff
@@ -48,6 +48,19 @@
     overflow: hidden; 
 }
 
+.simple-card .header-with-image {
```

Comment:
We might want to slap these logos on some existing cards too :) Let's do a follow-up to tackle that next sprint

Reply:
Sounds good, created a follow-up JIRA:)